pages/services/include/getting-started.tmpl

```diff
@@ -3,15 +3,15 @@ Getting started with a test instance of the DC/OS {{ model.techName }} service i
 ## Prerequisites
 
 - Depending on your security mode in Enterprise DC/OS, you may [need to provision a service account]({{ model.install.serviceAccountInstructionsUrl }}) before installing. Only someone with `superuser` permission can create the service account.
-	- `strict` [security mode](/latest/security/ent/#security-modes) requires a service account.
```

Comment:
Do not ever, ever edit a template (tmpl) file. These files are used by many different service docs, not just this one. I am deleting all these changes and moving the material you added to the getting-started/index.md file. They will appear at the beginning of the file, before you call the getting-started.tmpl file. I am reverting all the changes in this file back to the original.